services/QuillLMS/app/models/feedback_history.rb

```diff
@@ -46,9 +46,11 @@ class FeedbackHistory < ActiveRecord::Base
     OPINION = "opinion"
   ]
 
+  before_create :anonymize_activity_session_uid
   before_validation :confirm_prompt_type, on: :create
 
-  belongs_to :activity_session, foreign_key: :activity_session_uid, primary_key: :uid
+  belongs_to :activity_session_feedback_history, foreign_key: :activity_session_uid, primary_key: :feedback_history_session_uid
```

Comment:
I'm not sure if it will be reflected in this PR or a later one, but we'll need to change the name of this field to `feedback_session_uid` to avoid confusion down the line.